notebooks/en/index.md

```diff
@@ -12,6 +12,7 @@ Check out the recently added notebooks:
 - [Fine-tuning a Code LLM on Custom Code on a single GPU](fine_tuning_code_llm_on_single_gpu)
 - [RAG Evaluation Using Synthetic data and LLM-As-A-Judge](rag_evaluation)
 - [Advanced RAG on HuggingFace documentation using LangChain](advanced_rag)
+- [From OpenAI to Open LLMs with Messages API on Hugging Face](tgi_messages_api_demo)
```

Comment:
Since it'll be the latest added notebook, add it at the top of the list, please

Reply:
Good call! https://github.com/huggingface/cookbook/pull/28/commits/c9a853e5036b594b3fe89ecdeb81f97a07ba01cf